ddpui/management/commands/migrate_github_to_ui4t.py

```
@@ -0,0 +1,373 @@
+                sync_gitignore_contents(str(dbt_repo_dir))
+                self.stdout.write(self.style.SUCCESS("  .gitignore synced"))
+            else:
+                self.stdout.write(self.style.WARNING("  DRY-RUN: Would sync .gitignore"))
+
+            # Step 3: Initial commit
+            self.stdout.write("  Creating initial commit...")
+            if not dry_run:
+                try:
+                    # Add all files
+                    subprocess.run(
+                        ["git", "add", "."],
```

Ruff (0.14.13)

Starting a process with a partial executable path

(S607)